src/frontend/test_runner/tests/testdata/index.yaml

```diff
@@ -0,0 +1,15 @@
+- sql: |
+    create table t1 (v1 int, v2 float);
+    create table t2 (v3 int, v4 numeric, v5 bigint);
+    create index t1_v1 on t1(v1);
+    create index t2_v3 on t2(v3);
+    /* should generate delta join plan, and stream index scan (not supported for now) */
```

Comment:
So this plan is not the final result yet, right? What will be the final plan like?

Reply:
`StreamTableScan` will be replaced with `StreamIndexScan`. When converting to proto, it will still be `Chain` + `BatchPlan` + `Merge`, with an exception that `Chain` won't rearrange barriers (will be explained next week on meeting).